examples/multiple-daemons/run.rs

```diff
@@ -34,9 +34,13 @@ async fn main() -> eyre::Result<()> {
     build_dataflow(dataflow).await?;
 
     let (coordinator_events_tx, coordinator_events_rx) = mpsc::channel(1);
-    let (coordinator_port, coordinator) =
-        dora_coordinator::start(None, ReceiverStream::new(coordinator_events_rx)).await?;
-    let coordinator_addr = SocketAddr::new(Ipv4Addr::LOCALHOST.into(), coordinator_port);
+    let coordinator_bind = SocketAddr::new(
+        IpAddr::V4(Ipv4Addr::new(127, 0, 0, 1)),
+        DORA_COORDINATOR_PORT_DEFAULT,
+    );
+    let (coordinator_addr, coordinator) =
+        dora_coordinator::start(coordinator_bind, ReceiverStream::new(coordinator_events_rx))
+            .await?;
```

Comment:
So in this case, we're not covering the case of dora_coordinator using an external IP address.

Is it possible to use something like: https://github.com/haythem/public-ip
And try out public IPs for the coordinator?

Reply:
I'm surprised that actions runners might be accessible w/ a public ip. I think a better route would be to run the coordinator as a service container, completely separate from the daemons.

Reply:
Interesting idea! I agree that it would be useful to actually test external IPs on the CI if it's not too much effort.

Reply:
I opened https://github.com/dora-rs/dora/issues/475 to track this idea.